test/email_alert_api_test.rb

```diff
@@ -323,6 +323,40 @@
         end
       end
     end
+
+    describe "when content_id and links are provided" do
+      let(:content_id) { "fd427f55-7492-440d-ae3f-0fb6c3592b21" }
+      let(:slug) { "/my-new-list" }
```

Comment:
Not to be super nitpicky again (I am!) but `/my-new-list` isn't a slug either. It's a base path! 😱